src/js/app.js

```diff
@@ -19,8 +19,21 @@ const loadDefs = () => {
     })
 }
 
+function validateEmail(email) {
+  var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
On a sidenote, was this required because we had many spam? I always feel like very complex regex at some point endsup blocking an email you don't want to be blocked (especially devs who are experts and can have emails that are edge cases).

Just curious here